### PR TITLE
description for Input.is_mouse_button_pressed()

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by neikeq** [comment]
Nice. I don't think specifying the button values is required, since they are declared as constants in `@Global Scope`:

``` GDScript
BUTTON_LEFT = 1
BUTTON_RIGHT = 2
BUTTON_MIDDLE = 3
```

**Comment by lordadamson** [comment]
I actually didn't know that. Thanks! I modified the description.

**Comment by akien-mga** [comment]
Tip: you can use `[@Global Scope]` to make a hyperlink to the actual `@Global Scope` help.

Thus I would rephrase to:
```
Returns whether the given mouse button is pressed or not. You can use the BUTTON_* constants listed in [@Global Scope] to refer to the specific buttons.
```

Could you also merge all commits into one with `git rebase -i HEAD~3`, and replace "pick" by "fixup" for all commits apart from the first one?

**Comment by lordadamson** [comment]
yes sir. done.

**Comment by akien-mga** [comment]
Awesome :)
